frontend/packages/console-shared/src/utils/user-settings.ts

```diff
@@ -44,7 +44,7 @@ export const updateConfigMap = async (
   } catch (err) {
     // eslint-disable-next-line no-console
     console.error(err);
-    throw err;
+    return Promise.reject();
```

Comment:
I'm unclear on the intent of this change. Won't this be treated as a rejected promise either way?

Reply:
Yes they both return a rejected promise in an async function. Why the change?

Reply:
My bad, confused myself here. fixed it.